### DISCUSSION TITLE
Error during Build

### DISCUSSION THREAD

**Discussion by stssheltonrajkumar** [vikejs/vike#1651] [Discussion]
I using vite 0.4.171, I enable prerender: true, and i  using this option for all depedency as internal ssr: { noExternal: [/./] },
 during build prerender HTML is getting TypeError as _defineComponent is not a function, please provide a solution.
 
 thanks.

**Comment by brillout** [comment]
This doesn't seem related to Vike, but feel free to let me know if you think it does.
